mysql-test/suite/rocksdb/t/bulk_load_errors.test

```diff
@@ -0,0 +1,47 @@
+--source include/have_rocksdb.inc
+--source include/not_crashrep.inc
+
+--let SEARCH_FILE=$MYSQLTEST_VARDIR/tmp/rocksdb.bulk_load_errors.err
+--let $restart_parameters=restart:--log-error=$SEARCH_FILE
+--replace_result $SEARCH_FILE LOG_FILE
```

Comment:
A nit which does not require fixing, but something to consider in the future: I'd prefer replacing $MYSQLTEST_VARDIR with MYSQLTEST_VARDIR only, there is no need to hide the constant file path part